.github/workflows/devcontainer.yaml

```diff
@@ -43,25 +43,17 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - id: set-options
-        run: |
-          if [ "${{ github.ref }}" = "refs/heads/main" ]; then
-            echo "push=true" >>$GITHUB_OUTPUT
-          else
-            echo "push=false" >>$GITHUB_OUTPUT
-          fi
-
       - name: Build
         uses: docker/build-push-action@v4
         with:
           file: "./.devcontainer/base-image/Dockerfile"
           tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Does this push to all tags? That seems unusual (but no real downside)?

Reply:
I'm sorry, but I don't understand what your concern is.
I think it is common to specify multiple tags in `docker buildx build` command and push them at once.

Note that this is copied from the README of the docker metadata action.
https://github.com/docker/metadata-action
For now, only the `latest` raw tag is set, so the output is always `ghcr.io/prql/prql-devcontainer-base:latest`.
(No processing is done other than converting uppercase to lowercase!)
https://github.com/PRQL/prql/blob/e0d90d3a45fa5ca217c1e35a604d9ba9af0d7fbf/.github/workflows/devcontainer.yaml#L26-L31